src/helpers/navigationValidation.js

```diff
@@ -175,7 +176,7 @@ export function prevNextButtons(state) {
       ? { panel: "dataDisplayLabel", text: "Back" }
       : null;
     next = isPanelAvailable(state, "trainModel")
-      ? { panel: "trainModel", text: "Train" }
+      ? { panel: "trainModel", text: I18n.t('trainModelButtonText') }
```

Comment:
is this the only string we're trying to translate? or is this PR just to implement functionality?

Reply:
The goal of this PR is to just establish an i18n foundation. A followup PR will have the rest of the strings.